de-de/c-de.html.markdown

```diff
@@ -0,0 +1,880 @@
+---
+language: c
+filename: learnc-de.c
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+lang: de-de
+---
+
+Ach, C. Immer noch **die** Sprache für modernes High-Performance Computing.
+
+C ist wahrscheinlich die niedrigste Programmiersprache, welche die meisten
```

Comment:
Sprache mit dem niedrigsten Abstraktionsniveau?

Reply:
Sounds better. I have changed it.